sites/authentication/authentication-fr.html

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Please apply the similar content change as I suggested for the English equivalent page